system-test/common.ts

```diff
@@ -16,10 +16,11 @@ import * as uuid from 'uuid';
 import {Cluster} from '../src/cluster';
```

Comment:
nit - any PR that's **only** updating test code should have the "test:..." prefix rather than "feat"

Reply:
done